app/main.py

```diff
@@ -1,2 +1,72 @@
+from fractions import Fraction
+from typing import Hashable, Any
+
+
 class Dictionary:
-    pass
+
+    class Node:
+
+        def __init__(self, key: Hashable, value: Any, hash_key: int) -> None:
+            self.key = key
+            self.value = value
+            self.hash_key = hash_key
+
+    def __init__(self) -> None:
+        self.load_factor = Fraction(2, 3)
+        self.number_of_elements = 0
+        self.capacity = 8
+        self.size = 0
+        self.hash_table = [None] * self.capacity
```

Comment:
`self.hash_table: list[Dictionary.Node | None] = [None] * self.capacity` - adding annotation here will make type hints available in the code below